mmpose/core/evaluation/top_down_eval.py

```diff
@@ -91,6 +91,43 @@ def _get_max_preds(heatmaps):
     return preds, maxvals
 
 
+def get_max_preds_3d(heatmaps):
```

Comment:
I saw that `_get_max_preds` is not public. Will `get_max_preds_3d` be used outside?

Reply:
I have modify the codes and only use keypoints_from_heatmaps3d outside.

Reply:
If `get_max_preds_3d` is not designed to be used outside, then we can rename it to `_get_max_preds_3d`.

Reply:
Sorry looked at an old diff. Ignore the above comment :)